app/controllers/calendar_controller.rb

```diff
@@ -0,0 +1,4 @@
+class CalendarController < ApplicationController
```

Comment:
I dont think Calendar is a good name for  the controller, choose something more descriptive of what it is controlling, ie what will you be using it for?  crud on bookings? crud on laundry rooms?

Reply:
changed controller's name to Booking.